### DISCUSSION TITLE
"vm_memory_high_watermark_paging_ratio" not really configurable greater than 1.0

### DISCUSSION THREAD

**Discussion by morganteg** [rabbitmq/rabbitmq-server#7524] [Questions]
Hi,

RabbitMQ official documentation reports [here](https://www.rabbitmq.com/memory.html#memsup-paging:~:text=It%20is%20possible%20to%20set%20vm_memory_high_watermark_paging_ratio%20to%20a%20greater%20value%20than%201.0.%20In%20this%20case%20queues%20will%20not%20page%20their%20contents%20to%20disc.%20If%20this%20causes%20the%20memory%20alarm%20to%20go%20off%2C%20then%20producers%20will%20be%20blocked%20as%20explained%20above.):

> It is possible to set vm_memory_high_watermark_paging_ratio to a greater value than 1.0. In this case queues will not page their contents to disc. If this causes the memory alarm to go off, then producers will be blocked as explained above. 

I tried it on RabbitMQ v3.11.9 but it doesn't work with the following error message:

>  vm_memory_high_watermark_paging_ratio invalid, Float is not between 0 and 1

Is the documentation wrong? Any ideas? Thanks!

**Comment by michaelklishin** [comment]
The direction of QQs, streams and classic queues v2 has been to "always move data to disk early", so the above doc section is now obsolete.

**Comment by michaelklishin** [comment]
I have pushed a doc update.
